mne/inverse_sparse/_gamma_map.py

```diff
@@ -237,8 +236,8 @@ def gamma_map(evoked, forward, noise_cov, alpha, loose=0.2, depth=0.8,
 
     # make forward solution in fixed orientation if necessary
     if loose is None and not is_fixed_orient(forward):
-        forward = deepcopy(forward)
-        _to_fixed_ori(forward)
+        forward = convert_forward_solution(
+            forward, surf_ori=True, force_fixed=True, copy=True)
```

Comment:
This is an API change. It probably qualifies as a bugfix, but we need to document it since it could change people's results.